app/models/ticket.rb

```diff
@@ -1198,6 +1198,11 @@ def articles
     Ticket::Article.where(ticket_id: id).order(:created_at, :id)
   end
 
+  def self.by_state
+    pending_states = Ticket::State.by_category(:pending).map(&:id)
+    Ticket.where(state_id: pending_states).to_a
```

Comment:
Indexed: The database uses a "range scan" to read more than 200000 rows in tickets via the **index_tickets_on_state_id** index (state_id)
Results: The database returns 200000 row(s) to the client.
Estimated query time: 6.05s